### PR TITLE
Fix nginx module e2e and upgrade nginx

### PR DESCRIPTION
**Pull request by zaslavskii**

### CODE REVIEW

**Review by mou — APPROVED** [review]

**Review by Maxitosh — APPROVED** [review]

File: docker/Dockerfile.nginx [hunk 1]
```diff
@@ -7,12 +7,12 @@ WORKDIR /tmp
...
 COPY ./src /tmp/ngx_http_websocket_stat_module
-RUN cd ./nginx && \
+RUN cd ./chainstack && \
     ./configure  --with-compat \
     --add-dynamic-module=../ngx_http_websocket_stat_module && \
     make modules
 
-FROM k8s.gcr.io/ingress-nginx/controller:v0.44.0
-COPY --from=builder /tmp/nginx/objs/ngx_http_websocket_stat_module.so /etc/nginx/modules/
+FROM k8s.gcr.io/ingress-nginx/controller:v1.8.1
```
Comment (Dmitry1987) [inline docker/Dockerfile.nginx:17]: I used v1.8.2 , any chance that's why half of the fmt directives do not work for me?  if I add any of the 'remote_addr' / 'server_addr' / 'uri' / 'request' / etc'   the nginx will crash.  what is the version to use, is 1.8.1 the best option to make it work? i thought it'll be compatible with a few other nginx versions but seems like there are major differences 😅 to the point that some log parts won't work at all.   
I managed to make my log work with those settings 
```
    ws_log_format open "OPEN, SOURCE: $ws_packet_source (CODE $ws_opcode)";
    ws_log_format server "SERVER->CLIENT: $time_local | SOURCE: $ws_packet_source AGE: $ws_conn_age seconds, Size: $ws_message_size (CODE $ws_opcode)";
    ws_log_format client "CLIENT->SERVER: $time_local | SOURCE: $ws_packet_source AGE: $ws_conn_age seconds, Size: $ws_message_size (CODE $ws_opcode)";
    ws_log_format close "CLOSE, SOURCE: $ws_packet_source AGE: $ws_conn_age seconds (CODE $ws_opcode)";
```

but if i'll add any other directive from the available ones, it'll crash 🙈

Comment (Dmitry1987) [reply docker/Dockerfile.nginx:17]: btw thanks for maintaining this module, i'm baffled that there's no native support in nginx to debug websockets disconnets/issues 😵

Comment (mou) [reply docker/Dockerfile.nginx:17]: @zaslavskii this question is for you

Comment (zaslavskii) [reply docker/Dockerfile.nginx:17]: @Dmitry1987 thanks for the question! Let me take a look at the issue closer and get back to you

**Pull request merged**